templates/components/button.html.twig

```diff
@@ -1,8 +1,6 @@
 {% macro button(icon, values, type) %}
     {%- apply spaceless -%}
-        {% if type is same as (false) %}
-            {% set class = "" %}
-        {% elseif type is null %}
+        {% if type is null or type is same as (false) %}
```

Comment:
I've let `type is same as (false)` for you (@kevinpapst) in case you still have condition on that.

Need to be removed on the next major version + deprecated DOC

Reply:
may I ask why you changed that? this is on use-case less now

Reply:
In what use case you define `type` as false ?

Reply:
The system is highly dynamic and used in combination with by an Event driven approach, which is again used by plugins (which I did not create) of my software. So I tried to find the places by triggering an exception if `type` is false, but I am in the middle of a refactoring and have too many other test failures right now... 
I need to fix these other errors first and then I can answer the question if I need support for false or not.
Maybe that was only needed in AdminLTE but is not needed in Tabler anymore, which I so believe, but can't prove.